File: Math/Mirror Distance of an Integer/Mirror_Distance_of_an_Integer.java
//Problem Name: Mirror Distance of an Integer
//Problem Link: https://leetcode.com/problems/mirror-distance-of-an-integer/description/

class Solution {
    public int reverseNum(int num) {
        int reverseNum = 0;
        while (num > 0) {
            reverseNum = reverseNum * 10 + (num % 10);
            num /= 10;
        }
        return reverseNum;
    }

    public int mirrorDistance(int n) {
        return Math.abs(n - reverseNum(n));
    }
}

File: Math/Mirror Distance of an Integer/Mirror_Distance_of_an_Integer.cpp
//Problem Name: Mirror Distance of an Integer
//Problem Link: https://leetcode.com/problems/mirror-distance-of-an-integer/description/

class Solution {
public:
    int reverseNum(int n) {
        int num = 0;
        while (n > 0) {
            num = num * 10 + (n % 10);
            n /= 10;
        }
        return num;
    }

    int mirrorDistance(int n) { 
        return abs(n - reverseNum(n)); 
    }
};